prow/config/config.go

```diff
@@ -57,6 +57,8 @@ type Tide struct {
 	// These must be valid GitHub search queries. They should not overlap,
 	// which is to say two queries should never return the same PR.
 	Queries []string `json:"queries,omitempty"`
+	// EnableVerboseLog enables verbose logging for tide dynamically.
+	EnableVerboseLog bool `json:"enable_verbose_log,omitempty"`
```

Comment:
Do we want one "verbose_log" config option for all of prow, or an individual option for each component?

Reply:
Good question, I thought about this and don't feel strong in any way.

Single option:
- We don't bloat the config.
- Easy to use when you want to debug an event e2e.

Each component has its own:
- Finer-grained logging. We may want to run hook with high verbosity and jenkins-operator turned off.

I am willing to try out the single option now that I have layed out my thoughts on the pros of each side. Looking back at hook logging, it feels to me that high verbosity on it is not necessary by default.

Reply:
Hrm, re hook it's not possible to force it update its client w/o hacking in it a bit more so it's left on by default for now.